test/instrumentation/queue.js

```diff
@@ -28,3 +28,37 @@ test('queue flush isolation', function (t) {
   queue.add(1)
   queue.add(2)
 })
+
+test('queue flush callback success', function (t) {
+  var flush = 0
+  var queue = new Queue({maxQueueSize: 2}, function (arr, done) {
+    t.equal(arr.length, 1)
+    t.deepEqual(arr, [1])
+    flush++
+    done()
+  })
+
+  queue.add(1)
+  queue._flush(function (err) {
```

Comment:
Travis seems to be having problems, so it still hasn't run the tests, but according Jenkins, standard is complaining this error object isn't handled